packages/components/menu/src/menu-item.vue

```diff
@@ -65,7 +64,7 @@ isPropAbsent(props.index) &&
   debugWarn(COMPONENT_NAME, 'Missing required prop: "index"')
 
 const instance = getCurrentInstance()!
-const rootMenu = inject<MenuProvider>(MENU_INJECTION_KEY)
+const rootMenu = inject<MenuProvider>(MENU_INJECTION_KEY)!
 const nsMenu = useNamespace('menu')
 const nsMenuItem = useNamespace('menu-item')
 if (!rootMenu) throwError(COMPONENT_NAME, 'can not inject root menu')
```

Comment:
`rootMenu` is checkout by the judgment below.

```suggestion
const rootMenu = inject<MenuProvider>(MENU_INJECTION_KEY)
const nsMenu = useNamespace('menu')
const nsMenuItem = useNamespace('menu-item')
if (!rootMenu) throwError(COMPONENT_NAME, 'can not inject root menu')
```

Reply:
<img width="1319" height="795" alt="image" src="https://github.com/user-attachments/assets/bd89b52f-f845-446a-940c-77e5f7f261f6" />

Reply:
Oh it is used in template, ok we can revert the change.